tests/core/test_accounts.py

```diff
@@ -409,54 +427,81 @@ def test_eth_account_recover_transaction_from_eth_test(acct, transaction):
 
 
 @pytest.mark.parametrize(
-    'private_key, password, expected_decrypted_key',
+    'private_key, password, kdf, expected_decrypted_key, expected_kdf',
     [
         (
             web3js_key(),
             web3js_password(),
+            None,
             to_bytes(hexstr=web3js_key()),
+            'scrypt'
+        ),
+        (
+            web3js_key(),
+            web3js_password(),
+            'pbkdf2',
+            to_bytes(hexstr=web3js_key()),
+            'pbkdf2'
         ),
         (
             web3js_private_key(web3js_key()),
             web3js_password(),
+            None,
             web3js_private_key(web3js_key()).to_bytes(),
+            'scrypt'
         ),
     ],
-    ids=['hex_str', 'eth_keys.datatypes.PrivateKey']
+    ids=['hex_str', 'hex_str_provided_kdf', 'eth_keys.datatypes.PrivateKey']
 )
-def test_eth_account_encrypt(acct, private_key, password, expected_decrypted_key):
-    encrypted = acct.encrypt(private_key, password)
+def test_eth_account_encrypt(acct, private_key, password, kdf, expected_decrypted_key, expected_kdf):  # noqa: E501
```

Comment:
Are we fine with silencing this flake warning here and below or is there some pattern established for how we format method signatures that are too long per the linting?

Reply:
Generally we break the line rather than silencing, dropping each argument on it's own line like this: https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md#function-signatures

Reply:
Fixed here
https://github.com/ethereum/eth-account/pull/38/commits/8aec325d1b02c045143b0106e6a807524c04c6b6#diff-1de692d823115945c2232e1aa3066811L456
and here
https://github.com/ethereum/eth-account/pull/38/commits/8aec325d1b02c045143b0106e6a807524c04c6b6#diff-1de692d823115945c2232e1aa3066811L498